### ISSUE TITLE
Insufficient Contrast on 'Success' text

### ISSUE THREAD

**Issue by jlkravitz** [csg-org/CompactConnect#405]
Path: `/aslp/Settings`, `/Account`, and all other pages that show a "success" message after submitting a form.
Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.84:1. Recommendation: change text colour to #288737.

<img width="179" alt="image" src="https://github.com/user-attachments/assets/6298686c-c100-476c-a2fe-644a709ff1bc" />

_Originally posted by @jlkravitz in https://github.com/csg-org/CompactConnect/pull/401#discussion_r1890390301_
